release_notes.md

```diff
@@ -6,9 +6,12 @@
 ## Bug fixes
 - Fixed handling of function timeouts inside entity and activity functions and added tests
 - Skip constructor of AzureStorageDurabilityProvider if not used, to avoid spurious validation exceptions
+- Fixed stuck orchestration issue caused when CallEntityAsync was the first action in an orchestration and the entity completed before the orchestrator completed its first history checkpoint (fixed in DT.AzureStorage https://github.com/Azure/durabletask/pull/657)
```

Comment:
Were there any fixes in 1.10.0 that we should call out here?

Reply:
This seemed like the biggest change to me impacting the most customers. The other top contenders are the ttl update, a correlation compression bug, updated FetchInstanceStatus logs (and a few more that dont seem that important to DF customers). Do you think I should add these other updates?